### DISCUSSION TITLE
求教大佬们，，为什么无法运行

### DISCUSSION THREAD

**Discussion by ZhengchaoCao** [qd-today/qd#252] [Q&A] [answered]
按照主页介绍用docker安装的，可是安装后无法启动容器呀，，求教大佬们怎么处理了，，![Screenshot_20220427-185544.jpg](https://user-images.githubusercontent.com/39791246/165503775-6dd95941-15c3-426f-84e9-7fb41d4bac3c.jpg)

**Comment by a76yyyy** [comment, marked as answer]
怀疑是你把框架源码映射到容器外了,导致操作没有权限
